docs/autonomy.md

```diff
@@ -199,7 +199,6 @@ Usage: autonomy analyse abci check-app-specs [OPTIONS]
 Options:
   --check-all          Check all available definitions.
   --packages-dir PATH  Path to packages directory; Use with `--check-all` flag
-  --mermaid            Mermaid file.
```

Comment:
Do we no longer support this?

Reply:
We support yaml, json or mermaid for **output**, and yaml json for input. Thus the removal of this line here.